charts/gardener/gardenlet/values.yaml

```
@@ -159,7 +159,10 @@ global:
       #       namespace: istio-ingress-handler-2
       #       labels:
       #         istio: ingressgateway-handler-2
-
+      ETCDConfig:
+        EnableBackupCompaction: false
+        EventsThreshold: 1000000
+        ActiveDeadlineDuration: "3h"
```

```suggestion
        enableBackupCompaction: false
        eventsThreshold: 1000000
        activeDeadlineDuration: "3h"
```

However, I would redesign this API as follows:

```yaml
etcdConfig:
  backupCompaction:
    enabled: false
    eventsThreshold: 100000
    activeDeadlineDuration: 3h
```

Please adapt everything accordingly.

---

I wouldn’t like to add another layer. It will lead to more number of null check while using.

---

Fair enough, I think in this file in can also be easily refactored later if needed. Still, please make the names lower case

---

This comment is still not resolved.

---

Right. i took care this time